utils/mainnet-res/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+use near_account_id::AccountId;
+use near_chain_configs::Genesis;
+use near_primitives::receipt::ReceiptResult;
+
+pub fn mainnet_restored_receipts() -> ReceiptResult {
+    let data = include_bytes!("../res/mainnet_restored_receipts.json");
+    serde_json::from_slice(data)
+        .expect("File with receipts restored after apply_chunks fix has to be correct")
```

Comment:
Pity there’s no obvious way to parse the file at compile time.  `serde_json::json!` almost gives us that but we’re missing C’s `#include`.  Oh well…